sunspec/integrationpluginsunspec.cpp

```diff
@@ -1347,31 +1352,31 @@ void IntegrationPluginSunSpec::onStorageBlockUpdated()
 
     switch (storage->chaSt()) {
     case SunSpecStorageModel::ChastOff:
-        thing->setStateValue(sunspecStorageBatteryLevelStateTypeId, "Off");
+        thing->setStateValue(sunspecStorageStorageStatusStateTypeId, "Off");
```

Comment:
Do we really need this state? Seems quite redundant for little to no gain.

Reply:
one is for the interface, one is more informative for the user and for us if we see one in the field...